src/main/cpp/t02_shoes.cpp

```diff
@@ -28,10 +28,34 @@
 
 #include "t02_shoes.h"
 #include <iostream>
-
-
+#include <algorithm>
+#include <vector>
 using namespace std;
-
 int t02_shoes() {
-    
+    int c, n, count = 0;
+    cin >> c >> n;
+    vector <int> a(n);
+    for (int i = 0; i < n; i++) {
+        cin >> a[i];
+    }
+    sort(a.begin(), a.end());
+    int i = 0;
+    for(i; i < n and j; i++) {
+         bool j = true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'j' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4589232)